File: central.py
num1=float(input("Ingrese el primer numero: "))
num2=float(input("Ingrese el segundo numero: "))
num3=float(input("Ingrese el tercer numero: "))

if num1<num2 and num1>num3 or num1>num2 and num1<num3:
    central=num1
elif num2<num1 and num2>num3 or num2>num1 and num2<num3:
    central=num2
else:
    central=num3

print(f"El numero central es:{central} ")

File: Central.java
import java.util.Scanner;
public class Central {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Ingrese el primer numero: ");
        float num1= scanner.nextFloat();
        System.out.println("Ingrese el segundo numero: ");
        float num2= scanner.nextFloat();
        System.out.println("Ingrese el tercer numero: ");
        float num3= scanner.nextFloat();
        float central;

        if (num1<num2 && num1>num3 || num1>num2 && num1<num3){
            central = num1;
        } else if (num2 < num1 && num2 > num3 || num2 > num1 && num2 < num3) {
            central = num2;
        }else
            central = num3;

        System.out.printf("El numero central es:%n%.2f",central);
    }
}
